src/main.ts

```diff
@@ -324,38 +325,45 @@ export default function main() {
 
     userId = resolveUser(userId);
 
+    const existingAuth = getAuthToken(userId);
+    let channel = existingAuth?.channel;
+
     // while initializing, consider the channel active
     liveSatisfactionActive = true;
     try {
-      const res = await request(`${getUrl()}/feedback/prompting-init`, {
-        userId,
-      });
+      if (!channel) {
+        const res = await request(`${getUrl()}/feedback/prompting-init`, {
+          userId,
```

Comment:
we use this in the app to show "initialized status" right?
<img width="649" alt="Screenshot 2023-12-12 at 13 12 14" src="https://github.com/bucketco/bucket-tracking-sdk/assets/744/27f1a443-9797-4c55-a02e-f5755b0dea81">

do we need some kind of expiry for that to ensure an reasonably up-to-date initialized time?

Reply:
It is set to expire when the token's TTL expires (that is by default an hour). But even then, the initialization check only cares about at least one connection attempt, which is a prerequisite for anything being cached.
In either case, we're covered.

Reply:
who controls the TTL? We currently say "Last client activity" which could then be confusing since it might be up to an hour delayed?

Reply:
I see what you mean. I thought you were referring to the indicator of "installed". The last activity is based on a `prompting-auth` call. So if we want to cache this across pages we'd need to sacrifice a bit of precision.

We control the TTL in ingest (default is `60min`). We can set it to lower values, but that would increase the number of refresh requests from clients. I think `60m` is reasonable.

Reply:
got it. Maybe we can just adjust the display to say something more vague rather than the usual relative time? Like "Today" if it's within 24h and then regular relative time after that? the green color is still nice. I'd love to have that color on there still if it's "today"

Reply:
Certainly can be done, add some level of ambiguity as an option to those time cells. I'll look into it.